ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files
+ *
+ * To learn more see {
+ * @link http://symfony.com/doc/current/cookbook/bundles/extension.html#cookbook-bundles-extension-config-class
+ * }
+ */
+class Configuration implements ConfigurationInterface
```

Comment:
why an array for all type (node, node_collection, etc) and not one an arraynode (facade) and a scalarNode for each type ?